lib/shoryuken/manager.rb

```diff
@@ -99,22 +100,18 @@ def assign(queue, sqs_msg)
       end
     end
 
-    def rebalance_queue_weight!(queue)
-      watchdog('Manager#rebalance_queue_weight! died') do
-        if (original = original_queue_weight(queue)) > (current = current_queue_weight(queue))
-          logger.info "Increasing '#{queue}' weight to #{current + 1}, max: #{original}"
-
-          @queues << queue
-        end
+    def messages_present(queue)
+      watchdog('Manager#messages_present died') do
+        @polling_strategy.messages_present(queue)
       end
     end
 
-    def pause_queue!(queue)
-      return if !@queues.include?(queue) || Shoryuken.options[:delay].to_f <= 0
+    def queue_empty(queue)
```

Comment:
I can't see how to fix this. The method itself seems pretty straightforward to me.